index.js

```diff
@@ -10,8 +10,16 @@ module.exports = function (filepath, options) {
   var fullpath = (root) ? path.join(root, filepath) : filepath
 
   try {
-    return fs.statSync(fullpath).isFile()
-  } catch (e) {
-    return false
+    return fs.statSync(fullpath).isFile();
+  }
+  catch (e) {
+  
+    if (e.code == 'ENOENT') { // no such file or directory 
+      console.log("File does not exist.");
```

Comment:
Good idea. Can you explain why a console.log() here? That would produce noisy output to consumers. Perhaps just throw a new error?